js/custom.js

```diff
@@ -46,6 +46,16 @@ $(document).ready(function() {
 
 $(window).load(function() {
   $('#integrations_gif').attr('src', 'img/integrations.gif');
+
+  $.ajax({
+    type: "GET",
+    url: "http://fedsonslack.com/slack_api.php?method=users.count"
```

Comment:
We should come up with a front-end only approach for the API, maybe a .gitignored config or something